### ISSUE TITLE
License question

### ISSUE THREAD

**Issue by andrey-pr** [tatsu-lab/stanford_alpaca#132]
Hello.
LLAMA is custom-licensed product of meta, which distributed only for research purposes. Alpaca based on llama. Does it means that using of alpaca in commercial products is illegal?

**Comment by stefangrotz** [comment]
No, afaik all derivates from LLAMA have to use a non-commercial license too.

**Comment by jeffwadsworth** [comment]
From our Overlord Bing:  According to a Stanford announcement, Alpaca is intended only for academic research and any commercial use is prohibited. [This decision is based on three factors: First, Alpaca is based on LLaMA, which has a non-commercial license, so they necessarily inherit this decision](https://simonwillison.net/2023/Mar/13/alpaca/)[1](https://simonwillison.net/2023/Mar/13/alpaca/).

**Comment by andrey-pr** [comment]
> No, afaik all derivates from LLAMA have to use a non-commercial license too.

So its like GPL or I can not install even unmodified alpaka in my commercial project?

**Comment by andrey-pr** [comment]
> No, afaik all derivates from LLAMA have to use a non-commercial license too.

So its like GPL or I can not install even unmodified alpaka in my commercial project?

**Comment by Jermanis** [comment]
This repo is tagged as [Apache-2.0 license] (https://github.com/tatsu-lab/stanford_alpaca/blob/main/LICENSE)

**Comment by tytung2020** [comment]
> From our Overlord Bing: According to a Stanford announcement, Alpaca is intended only for academic research and any commercial use is prohibited. [This decision is based on three factors: First, Alpaca is based on LLaMA, which has a non-commercial license, so they necessarily inherit this decision](https://simonwillison.net/2023/Mar/13/alpaca/)[1](https://simonwillison.net/2023/Mar/13/alpaca/).

the license page shows Apache 2.0, which is a license allowing for commercial use:
https://github.com/tatsu-lab/stanford_alpaca/blob/main/LICENSE
I think it's a mistake and they should change it.
